hr_user_ciam_update/models/ciam_update.py

```diff
@@ -0,0 +1,34 @@
+from odoo import models, fields, api, _
+import json
+
+
+class CIAMUpdate(models.TransientModel):
+    _name = 'hr.employee.user.ciam'
+
+    employee_id = fields.Many2one('hr.employee', default=lambda self: self.env.context.get('active_id'))
+
+    def action_update_ciam(self):
+        ciam_id = self.env['ciam.client.config'].search([], limit=1)
+        if ciam_id:
+            headers = ciam_id.get_headers()
+            print(headers)
+
+            # Add User
+            querystring = {"client_secret": ciam_id.client_secret,
+                           "client_id": ciam_id.client_id}
+            payload = {'personNr': 1955010127777,
+                       'firstName': 'Pelle',
+                       'lastName': 'Svensson',
+                       'eMail': 'pelle.svensson@bolaget.se',
+                       'orgId': 'c4c0b8c2-dccc-4580-b2f9-88b4aeb9bc06',
+                       'username': 'pelle.svensson.14',
+                       'password': '123',
+                       'customerNr': '444', 'status': "1"}
+
+            url = ciam_id.get_url('v1/user/add')
+            response = self.request_call(method="POST",
+                                         url=url,
+                                         payload=json.dumps(payload),
+                                         headers=ciam_id.get_headers(),
+                                         params=querystring)
```

Comment:
Hi @iammiracle 

I tried to do the action, but an error occured. 
My guess is that `self.request_call` should be `ciam_id.request_call`.

Here is the traceback
```python
Error:
Odoo Server Error

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 656, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 314, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/usr/lib/python3/dist-packages/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 698, in dispatch
    result = self._call_function(**self.params)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 346, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/odoo/service/model.py", line 98, in wrapper
    return f(dbname, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 339, in checked_call
    result = self.endpoint(*a, **kw)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 941, in __call__
    return self.method(*args, **kw)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 519, in response_wrap
    response = f(*args, **kw)
  File "/usr/lib/python3/dist-packages/odoo/addons/web/controllers/main.py", line 966, in call_button
    action = self._call_kw(model, method, args, {})
  File "/usr/lib/python3/dist-packages/odoo/addons/web/controllers/main.py", line 954, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/usr/lib/python3/dist-packages/odoo/api.py", line 759, in call_kw
    return _call_kw_multi(method, model, args, kwargs)
  File "/usr/lib/python3/dist-packages/odoo/api.py", line 746, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/usr/share/odoo-hr/hr_user_ciam_update/models/ciam_update.py", line 29, in action_update_ciam
    response = self.request_call(method="POST",
AttributeError: 'hr.employee.user.ciam' object has no attribute 'request_call'
```

Reply:
Oh, I am just seeing this, it is ciam_id, thanks for point that out